packages/govtnz-ds-website/src/components/header.tsx

```diff
@@ -19,7 +19,7 @@ const Header = ({ siteTitle }: Props) => (
       <Container width="fixed">
         <Row>
           <Column xs="10" sm="10" md="8" lg="8">
-            <LogoLockUp siteTitle={siteTitle} />
+            <LogoLockUp />
```

Comment:
@holloway  I removed this site title it has an error and also isn't  a prop on the `LogoLockUp `.